src/lib/kclHelpers.ts

```diff
@@ -11,10 +11,7 @@ const DUMMY_VARIABLE_NAME = '__result__'
  * Calculate the value of the KCL expression,
  * given the value and the variables that are available
  */
-export async function getCalculatedKclExpressionValue(
-  value: string,
-  variables: VariableMap
-) {
+export async function getCalculatedKclExpressionValue(value: string) {
```

Comment:
So how is this still able to evaluate variables? Is it because they're in the execution cache and the mock execution uses that?

Reply:
Yep, mock execution has access to the memory from previous (non-mock) executions